resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -44,7 +44,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
   private val EXECUTOR_MODIFICATION_LOCK = new Object
   private val runningExecutorPods = new scala.collection.mutable.HashMap[String, Pod]
 
-  private val kubernetesMaster = Client.resolveK8sMaster(sc.master)
+  private val kubernetesMaster = "https://kubernetes"
```

Comment:
is this a special value to fabric8 which means to use the locally-configured k8s apiserver from within the pod?